lib/datadog/core/environment/process.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require_relative 'ext'
+require_relative '../normalizer'
+
+module Datadog
+  module Core
+    module Environment
+      # Retrieves process level information
```

Comment:
Add some very high level language regarding when the information in this module is useful.

Reply:
Addressed in https://github.com/DataDog/dd-trace-rb/pull/5033/commits/ec1e930447b7be4ee9eeb7bb9f6608a11ede3bc0